### PR TITLE
registering demo-participant

### PR DESCRIPTION
**Pull request by priyanshuvaya**
Excited to be a part of this.

### CODE REVIEW

File: content/participants/ttjc@priyanshuvaya/index.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+I have played Asian Chess Championship quite a few years ago (when i was in school) and also played in college.
+
+### help I need?
+
+I am learning to expand my knowledge more on whole MERN stack.
+
+### how to contact me?
+
+- LinkedIn
```
Comment (gkanishk) [inline content/participants/ttjc@priyanshuvaya/index.md:33]: Please add your social links

File: content/participants/ttjc@priyanshuvaya/index.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+type: "participant"
+name: "Priyanshu Vaya"
+portfolio: ""
+github-link: "https://github.com/priyanshuvaya"
+linkedin-link: "https://linkedin.com/in/priyanshuvaya"
+looking-for: "Job"
+description: "Good at React | Express | Javascript | Java | Csharp. Working as Software Developer"
+---
+
+Software Developer
```
Comment (gkanishk) [inline content/participants/ttjc@priyanshuvaya/index.md:12]: Write a bit more

File: content/participants/ttjc@priyanshuvaya/index.md (hunk 1 above)
Comment (priyanshuvaya) [reply content/participants/ttjc@priyanshuvaya/index.md]: done

File: content/participants/ttjc@priyanshuvaya/index.md (hunk 2 above)
Comment (priyanshuvaya) [reply content/participants/ttjc@priyanshuvaya/index.md]: done

**Comment by priyanshuvaya** [comment]
Made the changes

File: content/participants/ttjc@priyanshuvaya/index.md (hunk 1 above)
Comment (gkanishk) [reply content/participants/ttjc@priyanshuvaya/index.md]: can't see your changes done please add social links

**Pull request merged**